.github/workflows/build.yml

```diff
@@ -25,4 +25,22 @@ jobs:
           DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
         run: |
           echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
-          docker push antrea/ipfix-collector:latest
\ No newline at end of file
+          docker push antrea/ipfix-collector:latest
+  upload-kafka-images:
```

Comment:
can we make this job daily instead of doing it for every push on main branch? These are not dependent on our code right?

Reply:
@zyiou I am chatting with Antonin. He suggested using a version tag instead of latest. That means we can directly upload the image directly to Antrea repo when the version change is required. In this case, we do not need any job.

I think the hardcoded version for images work here. Do you see any issue?

Reply:
The reason is that there are CDN issues when using latest as the tag name.

Reply:
Got it. I was trying with Github action schedule function. Will change back since it is no longer needed. Thanks